dafni_cli/models/model.py

```diff
@@ -222,40 +230,55 @@ def metadata(self) -> ModelMetadata:
         )
         return self._metadata
 
-    def output_details(self, long: bool = False):
-        """Prints relevant model attributes to command line
+    def get_brief_details(self) -> List:
+        """Returns an array containing brief details about this model for
+        the get models command
 
-        Args:
-            long (bool): Whether to print with the (potentially long)
-                         description (ignored if description is None)
+        Returns
+            List: Containing display_name, model_id, status, access,
+                  publication date and summary
         """
+        return [
+            self.metadata.display_name,
+            self.model_id,
+            self.metadata.status,
```

Comment:
Can we convert this status into something that makes more sense when read by the users?

Reply:
Sure I have gone with the comment I wrote before, so that
```
            P -> Pending
            F -> Failed
            L -> Live
            S -> Superseded
            D -> Deprecated
```